xml/html_title.xml

```diff
@@ -2492,7 +2526,7 @@
     <param pos="0" name="os.device" value="Lights Out Management"/>
     <param pos="0" name="os.vendor" value="Oracle"/>
     <param pos="0" name="os.family" value="ILOM"/>
-    <param pos="0" name="os.product" value="Integrated Lights Out Manager firmware"/>
+    <param pos="0" name="os.product" value="ILOM"/>
```

Comment:
@hdm  - Returning this value to what it was earlier this week. You'd asked about the need to add ` firmware` to the end of the value in that PR.  I didn't like it at the time but I didn't have decent options.  With the current PR this is now trivial.